source/game.ingameinterface.bmx

```diff
@@ -1216,7 +1216,8 @@ endrem
 						'three seats - use only one left
 						If newCouchPositions[1] = intendedPosition Or newCouchPositions[0] = intendedPosition
 							intendedPosition = intendedPosition - 60
-							If newCouchPositions[1] = intendedPosition Or newCouchPositions[0] = intendedPosition Then newCouchPositions[spot] = intendedPosition - 60
+							If newCouchPositions[1] = intendedPosition Or newCouchPositions[0] = intendedPosition Then intendedPosition = intendedPosition - 60
+							newCouchPositions[spot] = intendedPosition
```

Comment:
Nur fuers Verstaendnis:

vorher:
Das Setzen von "intendedPosition" war uebersehen worden.
newCouchPositions[spot] wurde immer gesetzt, sobald die aeusseren und inneren Bedingungen zutrafen.

Jetzt:
"intendedPosition" wird gespeichert, newCouchPositions[spot] wird gesetzt, sobald die aeusseren Bedingungen zutreffen.

Reply:
vorher: intendedPosition war richtig berechnet worden (Zeile 1218), aber nur (nochmal angepasst und) verwendet wenn auch die zweite Bedingung erfüllt war (wenn es nach der ersten Korrektur noch eine Kollision gab)

nun: intendedPosition wird berechnet und dann in beiden Kollisionsfällen gesetzt.